modules/openapi-generator/src/main/resources/htmlDocs2/index.mustache

```diff
@@ -416,7 +441,10 @@
                                     <script>
                                       $(document).ready(function() {
                                         var schemaWrapper = {{{jsonSchema}}};
-                                        var schema = schemaWrapper.schema;
+                                        var schema = findNode('schema',schemaWrapper).schema;
+                                        if (!schema) {
+                                            schema = schemaWrapper.schema;
+                                        }
                                         if (schema.$ref != null) {
                                           schema = defsParser.$refs.get(schema.$ref);
                                         } else {
```

Comment:
This does not cover array type, e.g.:
```yaml
schema:
  type: array
  items:
    ref: '#/components/schemas/Pet'
```

So I had to add one more `if` statement:
```javascript
...
} else if (schema.type === 'array' && schema.items != null && schema.items.$ref != null) {
    schema.items = defsParser.$refs.get(schema.items.$ref);
} else {
    // line 450
}
```

Reply:
You're totally right. I tested it on two different versions of the OpenApi generator and on the older one there was no issue with arrays, so I didn't cover it up, but that's a great change.

Reply:
I've applied the suggested fix to this branch.